constants.js

```diff
@@ -0,0 +1,268 @@
+// Texts to be displayed in the Draw/Deal button
+const TXT_DRAW = 'Draw';
```

Comment:
Great work with constants and organising them into logical groups!

Reply:
Thanks Kai